uaclient/entitlements/fips.py

```diff
@@ -12,6 +12,74 @@
 event = event_logger.get_event_logger()
 
 
+FIPS_CONDITIONAL_PACKAGES = {
+    "xenial": [
+        "strongswan",
+        "strongswan-hmac",
+        "openssh-client",
+        "openssh-client-hmac",
+        "openssh-server",
+        "openssh-server-hmac",
+    ],
+    "bionic": [
+        "strongswan",
+        "strongswan-hmac",
+        "openssh-client",
+        "openssh-client-hmac",
+        "openssh-server",
+        "openssh-server-hmac",
+    ],
+    "focal": [
+        "strongswan",
+        "strongswan-hmac",
+        "openssh-client",
+        "openssh-server",
+    ],
+}
+# In containers, we don't install the ubuntu-fips
+# metapackage, but we do want to auto-upgrade any
+# fips related packages that are already installed.
+# These lists need to be kept up to date with the
+# Depends of ubuntu-fips.
+FIPS_CONTAINER_CONDITIONAL_PACKAGES = {
+    "xenial": [
+        "strongswan",
+        "strongswan-hmac",
+        "openssh-client",
+        "openssh-client-hmac",
+        "openssh-server",
+        "openssh-server-hmac",
```

Comment:
If we have those variables in auxiliary one, we can just reference it on all the other series add just the specific packages on each one of them